docs/rfc-1-staking-contract.adoc

```diff
@@ -18,18 +18,15 @@ not require any owner’s keys at all. The stake delegation should also optimize
 network throughput without compromising the security of the owners’ stake.
 
 This proposal aims at implementing a minimum viable staking contract version
-allowing to support legacy KEEP and NU stakes, as well as native T delegations
-in all applications developed against this staking contract version.
-It means that all stakers, no matter of the type of their stake (legacy KEEP,
-legacy NU, liquid T) will be able to participate in all applications developed
-against this staking contract version on equal rules.
+allowing to support native T delegations in all applications developed against 
+this staking contract version. It means that stakers will be able to participate 
+in all applications developed against this staking contract version on equal rules.
```

Comment:
```suggestion
this staking contract version.
```
This line was relevant in the context of different types of stakers (legacy vs native), but it can be removed now.